src/physics/autompo/opsum_to_mpo.jl

```diff
@@ -1,13 +1,11 @@
-function svdMPO(os::OpSum{C}, sites; kwargs...)::MPO where {C}
+function svdMPO(ValType::Type{<:Number}, os::OpSum{C}, sites; kwargs...)::MPO where {C}
```

Comment:
```suggestion
# `ValType::Type{<:Number}` is used instead of `ValType::Type` for efficiency, possibly due to increased method specialization.
# See https://github.com/ITensor/ITensors.jl/pull/1183.
function svdMPO(ValType::Type{<:Number}, os::OpSum{C}, sites; kwargs...)::MPO where {C}
```